src/main/kotlin/net/axay/kspigot/gui/GUIHolder.kt

```diff
@@ -77,7 +77,7 @@ object GUIHolder : AutoCloseable {
 }
 
 private val InventoryAction.isGUIClick
-    get() = this == InventoryAction.PICKUP_ALL || this == InventoryAction.PICKUP_HALF
+    get() = this == InventoryAction.PICKUP_ALL || this == InventoryAction.PICKUP_HALF || this == InventoryAction.PICKUP_SOME || this == InventoryAction.PICKUP_ONE || this == InventoryAction.MOVE_TO_OTHER_INVENTORY
```

Comment:
Could you also add a new line after this extension value? This broke due to my reformat commit.